From 7b0ca20d53b80cddd9afe9d4a84cb0978d10f161 Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynh.nguyen.xb@renesas.com>
Date: Tue, 14 Jul 2020 13:03:59 +0700
Subject: [PATCH 110/433] ARM: dts: iwg23s-sbc: spi: Add sst25vf016b spi flash
 support

RZ/G1C is compatible with sst25vf016b SPI flash. This patch adds support
for it.

Signed-off-by: khanhluu <khanh.luu.xw@rvc.renesas.com>
Signed-off-by: Quynh Nguyen <quynh.nguyen.xb@renesas.com>
---
 arch/arm/boot/dts/r8a77470-iwg23s-sbc.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/r8a77470-iwg23s-sbc.dts b/arch/arm/boot/dts/r8a77470-iwg23s-sbc.dts
index 43fd91f..e8a917b 100644
--- a/arch/arm/boot/dts/r8a77470-iwg23s-sbc.dts
+++ b/arch/arm/boot/dts/r8a77470-iwg23s-sbc.dts
@@ -187,9 +187,9 @@
 	flash: flash@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "issi,is25lp016d", "jedec,spi-nor";
+		compatible = "sst,sst25vf016b", "issi,is25lp016d", "jedec,spi-nor";
 		reg = <0>;
-		spi-max-frequency = <133000000>;
+		spi-max-frequency = <50000000>;
 		spi-tx-bus-width = <1>;
 		spi-rx-bus-width = <1>;
 		m25p,fast-read;
-- 
2.7.4

